SwiftLeeds/Views/Tab/Tabs.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Tabs.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 25/06/2022.
+//
+
+import SwiftUI
+
+struct Tabs: View {
+    init() {
+        UITabBar.appearance().backgroundColor = UIColor(named: "TabBarBackground")
```

Comment:
Given lifecycle in SwiftUI, I generally find it's best to have an object ( say UIAppearanceConfigurator ) which does everything like this, UINavigationBar etc, inside your top level scene delegate or "App".

So like this maybe:

```suggestion
class UIAppearanceConfigurator {
  func configure() {
    configureTabBar()
    configureNavBar()
   }

   private func configure...
}
```

Reply:
I'll move the initialiser to App, but don't see the point in overcomplicating things at this stage as there's only currently the one appearance.

Reply:
I think as we're using pre iOS 16 we know we'll need a bunch more of these, so its worth considering now imo, but happy to leave it until later - just watch for similar lines springing up allover the place